docs/ota-client-guide/modules/ROOT/pages/_partials/recommended-steps.adoc

```diff
@@ -115,3 +115,12 @@ Once you understand how the libaktualizr demo application works, you'll want to
 After you've finished integration and development, you'll need to move disk images from one account to another. For example, you might want to send a development build that you’re happy with to the QA team, or send that build to the deployment team once it’s passed QA.
 
 //  end::deploy-steps[]
+
+//  tag::firstbuild-nextstep[]
+[TIP]
+====
+Once you've build your first image, try adding some new software and xref:latest@ota-client::pushing-updates.adoc[push an update to the OTA Connect server].
```

Comment:
Is there some gotcha about having to use the extended xref syntax in partials? Shouldn't this work with just xref:pushingupdates.adoc[]?

Reply:
This content is included in getstarted page so xref:pushingupdates.adoc[]? shows up as broken when I view the page there.
i.e: https://docs.ota.here.com/getstarted/dev/raspberry-pi.html
Includes content from
https://docs.ota.here.com/ota-client/latest/build-raspberry.html
the the xref has to work in both places

Reply:
Interesting, I would've hoped Antora was smart enough to resolve it correctly, even in includes, since it has all the information it needs at include time.

Does it also fail without the `latest@`?

Reply:
> Does it also fail without the `latest@`?

Nope, it works. Looks like just the component is needed,